triton-server/src/notification.ts

```diff
@@ -4,40 +4,73 @@ import * as email from "./contact-service"
 import { getFreezeManAuthenticatedAPI } from "./freezeman/api"
 import { defaultDatabaseActions } from "./download/actions"
 import { logger } from "./logger"
-import { Dataset } from "./freezeman/models"
+import { Dataset, FreezemanUser, ValidationFlag } from "./freezeman/models"
 
 export const start = () => {
-    const cronExpression = "0 * * * *"
+    const cronExpression = "*/1 * * * *"
     logger.info(`Notification service started to run. (${cronExpression})`)
     const task = cron.schedule(cronExpression, async () => {
         logger.info("Executing notification service.")
+        await sendLatestReleasedNotificationEmail()
+        await sendDatasetValidationStatusUpdateEmail()
+    })
 
-        const db = await defaultDatabaseActions()
-        // this db action fails silently if the table does not exist
-        const lastReleaseDate = (await db.getLatestReleaseNotificationDate())
-            .last_released_notification_date
+    return () => {
+        task.stop()
+    }
+}
 
-        const freezemanApi = await getFreezeManAuthenticatedAPI()
+export const sendDatasetValidationStatusUpdateEmail = async () => {
+    const db = await defaultDatabaseActions()
 
-        const releasedDatasets = (
-            await freezemanApi.Dataset.listByReleasedUpdates(lastReleaseDate)
-        ).data.results.map((dataset) => ({ ...dataset }))
+    const freezemanApi = await getFreezeManAuthenticatedAPI()
 
-        logger.debug(
-            `Found ${releasedDatasets.length} datasets to potentially notify for release.`,
+    const lastValidationStatusUpdate = (
+        await db.getLatestValidatedNotificationDate()
+    ).last_validated_notification_date
+
+    const validatedDatasets = (
+        await freezemanApi.Dataset.listByValidatedStatusUpdates(
+            lastValidationStatusUpdate,
         )
-        if (releasedDatasets.length > 0) {
-            await sendNotificationEmail(releasedDatasets)
-        }
-    })
+    ).data.results.map((dataset) => ({ ...dataset }))
 
-    return () => {
-        task.stop()
+    logger.debug(
+        `Found ${validatedDatasets.length} datasets to potentially notify for release.`,
+    )
+    // the email portion of the logic
+    if (validatedDatasets.length > 0) {
+        let formattedData: ExtractedValidatedNotificationData[] =
+            await extractValidatedDatasetsInfo(validatedDatasets)
+        datasetObjectTestEmail(formattedData)
```

Comment:
shouldn't you be sending real email and not with `datasetObjectTestEmail`?

Reply:
i first build with the test email since i can only test locally :)